doc: CI system link added, clarity increased (#20215)
bitcoin/bitcoin: README.md

> in Python, that are run automatically on the build server.

Reviewer: ```suggestion
in Python.
```

Author: commit done

These tests can be run (if the [test dependencies](/test) are installed) with: `test/functional/test_runner.py`

> The [Travis CI system](https://travis-ci.org/) makes sure that every pull request is built for Windows, Linux, and macOS, and that unit/sanity tests are run automatically.

Reviewer: ```suggestion
The [CI system](/ci) defines build configs for Windows, Linux, and macOS, unit/sanity tests are part of the CI configs.
```

Author: commit done
